<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDownloadManager</class>
 <widget class="QDialog" name="CDownloadManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download manager - JPReader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableView" name="tableDownloads">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelProgress">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelWriter"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonClearDownloaded">
           <property name="toolTip">
            <string>Leave failed downloads</string>
           </property>
           <property name="text">
            <string>Clear downloaded</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonClearFinished">
           <property name="text">
            <string>Clear finished</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonAbortActive">
           <property name="text">
            <string>Abort active</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonAbortAll">
           <property name="text">
            <string>Abort all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Limit downloads</string>
           </property>
           <property name="buddy">
            <cstring>spinLimitCount</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinLimitCount">
           <property name="specialValueText">
            <string>Unlimited</string>
           </property>
           <property name="suffix">
            <string> threads</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonHide">
           <property name="text">
            <string>Hide</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableDownloads</tabstop>
  <tabstop>buttonClearDownloaded</tabstop>
  <tabstop>buttonClearFinished</tabstop>
  <tabstop>buttonAbortActive</tabstop>
  <tabstop>buttonAbortAll</tabstop>
  <tabstop>spinLimitCount</tabstop>
  <tabstop>buttonHide</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonHide</sender>
   <signal>clicked()</signal>
   <receiver>CDownloadManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
